src/lib/viewers/doc/_docBase.scss

```diff
@@ -113,7 +113,8 @@ $thumbnail-sidebar-width: 226px;
     @include bp-breakpoint-medium {
         $bp-thumbnails-animation: 300ms cubic-bezier(.4, 0, .2, 1);
 
-        &.bp-loaded {
+        &.bp-loaded.bp-thumbnails-open-active,
```

Comment:
Now we'll only apply the transition to bp content if the active classes are present. Those classes only get added in the user generated toggle case.